src/pages/ReactionPage/ReactionPage.tsx

```diff
@@ -17,9 +18,8 @@ const ReactionPage = () => {
   const partygoers = useSelector(partygoersSelector);
   const usersById = partygoers;
   const reactions = useSelector((state) => state.firestore.ordered.reactions);
-  const chats = useSelector((state) =>
-    state.firestore.ordered.venueChats?.filter((chat) => chat.deleted !== true)
-  );
+  const chats = useVenueChats();
+  const filteredChats = chats?.filter((chat) => chat.deleted !== true);
 
   useFirestoreConnect([
```

Comment:
Technically this is checking for `venue` currently, but then uses `venue.name` in the `doc`. So we should make the check `venue?.name`, otherwise it has the (rare) chance to end up `undefined` and trigger a pathological query

Reply:
resolved in 354167a550c42e93a27cc5c1e431145ff1caeef7